lib/commands/init.js

```diff
@@ -9,12 +9,16 @@ const modifyHelper = require("../utils/modify-config-helper");
  * First function to be called after running the init flag. This is a check,
  * if we are running the init command with no arguments or if we got dependencies
  *
- * @param {Object} pkg - packages included when running the init command
+ * @param {Array} args - array of arguments such as
+ * packages included when running the init command
  * @returns {Function} creator/npmPackagesExists - returns an installation of the package,
  * followed up with a yeoman instance of that if there's packages. If not, it creates a defaultGenerator
  */
 
-module.exports = function initializeInquirer(pkg) {
+module.exports = function initializeInquirer(...args) {
+	const popArgs = args ? args.slice(2).pop() : null;
```

Comment:
this means, `popArgs` ->  Do you mean you want to pop the arguments right. ( more than one)
The 3rd argument will always have all the packages that you want to check or the packages are in argument 3, 4, 5?